LibGAP.jl/src/gap_to_julia.jl

```diff
@@ -101,11 +101,15 @@ gap_to_julia(type_obj,obj,recursion_dict) = gap_to_julia(type_obj,obj)
 
 ## Convert GAP string to Array{UInt8,1}
 function gap_to_julia( ::Type{Array{UInt8,1}}, obj :: MPtr )
-    if ! Globals.IsStringRep( obj )
-        return Array{UInt8,1}(gap_to_julia(Array{UInt16,1},obj))
+    ## convert strings to uint8 lists, if requested
+    if Globals.IsStringRep( obj )
+        array = UNSAFE_CSTR_STRING( obj )
+        return deepcopy(array)
```

Comment:
Man, this code formatting really is all over the place :/. I really wish we had something like `clang-format` for Julia code.